docs/libpod.conf.5.md

```diff
@@ -45,8 +45,24 @@ libpod to manage containers.
 **cni_plugin_dir**=""
   Directories where CNI plugin binaries may be located
 
+**pause_image** = ""
+  Pause container image name for pod pause containers.  When running a pod, we
+  start a `pause` processes in a container to hold open the namespaces associated with the
```

Comment:
Singular 'process'?

Reply:
FIxed

Reply:
My head still spins over "pod pause containers".  Does this work?
"The image name to use for the containers in the pod pause command."  or perhaps "... {running || controlling} the pod pause command."

Reply:
Sometimes this is called the infracontainer.  Perhaps we should change the lingo now to clear this up.